txircd/modules/rfc/cmd_away.py

```diff
@@ -16,6 +17,20 @@ def actions(self):
 		return [ ("commandextra-PRIVMSG", 10, self.notifyAway),
 				("commandextra-NOTICE", 10, self.notifyAway),
 				("extrawhois", 10, self.addWhois) ]
+
+	def load(self):
+		self.rehash()
+
+	def rehash(self):
+		self.ircd.isupport_tokens["AWAYLEN"] = self.ircd.config.get("away_length", 200)
```

Comment:
If we're going to do THIS, we need to document somewhere for non-core modules that it MUST be removed on unload.